Makefile

```diff
@@ -166,7 +166,9 @@ kevm_lemmas := infinite-gas.k                           \
 
 lemma_includes := $(patsubst %, $(KEVM_INCLUDE)/kframework/lemmas/%, $(kevm_lemmas))
 
-includes := $(patsubst %, $(KEVM_INCLUDE)/kframework/%, $(kevm_includes)) $(plugin_includes) $(lemma_includes)
+kevm_includes := $(patsubst %, $(KEVM_INCLUDE)/kframework/%, $(kevm_files))
+
+includes := $(kevm_includes) $(lemma_includes) $(plugin_includes) $(plugin_c_includes)
```

Comment:
It doesn't look like `$(plugin_c_includes)` was _removed_ anywhere, so just want to double check that it was indeed supposed to be added here.

Reply:
Yeah, the `$(includes)` variable is just used to make sure that we re-copy the `kevm` script into the build directory whenever one of the include files changes. This is because the `kevm` script is supposed to be able to enumerate those files for various things.

Basically, the `$(includes)` isn't used for much, so I'm just putting all the includes in there.